### PR TITLE
Mood background, fixes #7

### LINKED ISSUE #7: Change the background depending on the mood
The current background is static, it would be nice if the background could change when a mood is selected.

This would make the mood more visual and could influence the user experience.

### PR DESCRIPTION
**Pull request by Jaapp-**
Transitions into a fitting background when choosing a mood.

### CODE REVIEW

**Comment by JWGmeligMeyling** [comment]
Looks good! :+1:

**Comment by Azkle** [comment]
The background and this feature is very nice.

If you clicked on a mood and we want to load another background.
It's a bit slow and you see a grey page for a few seconds, then the good background loads. That's very ugly, but I think that's outside the scope of this feature.
Might be something to look into, in the future.

File: app/scripts/controllers/main.js [hunk 1]
```diff
@@ -41,6 +41,11 @@ angular.module('moodCatApp')
     $scope.moods = ['Angry', 'Nervous', 'Exciting', 'Happy', 'Pleasing', 'Relaxing',
       'Peaceful', 'Calm', 'Sleepy', 'Bored', 'Sad'];
 
+	$scope.chooseMood = function(mood) {
+		console.log("Choosemood called: " + mood);
```
Comment (Azkle) [inline app/scripts/controllers/main.js]: Please use the anuglar logger with $log instead of console.log.

File: app/scripts/controllers/main.js [hunk 2]
```diff
@@ -41,6 +41,11 @@ angular.module('moodCatApp')
...
+		angular.element("body").css("background-image", "url(http://moodcat.me/mood-bg/" + mood.toLowerCase() + ")");
```
Comment (JeremybellEU) [inline app/scripts/controllers/main.js]: Shouldn't we make a class of this? Then you can set the class of the object and in CSS declare the correct url. @GijsWeterings

File: app/views/moodSelection.html [hunk 3]
```diff
@@ -3,7 +3,7 @@
   <div ui-view class="row">
     <div>
       <h4>In the mood?</h4>
-      <btn-mood ui-sref="selectRoom" ng-model="blabla" ng-repeat="mood in moods">{{mood}}</btn-mood>
+	  <btn-mood ui-sref="selectRoom" ng-model="blabla" ng-repeat="mood in moods" ng-click="chooseMood(mood)">{{mood}}</btn-mood>
```
Comment (JeremybellEU) [inline app/views/moodSelection.html]: Indentation should be reverted as it was correct.

**Comment by JeremybellEU** [comment]
2 minor remarks. for the other part LGTM

**Pull request merged**